main.py

```diff
@@ -70,6 +70,8 @@ def get_config():
                         help='Momentum factor for SGD only (default: %(default)s)')
     parser.add_argument('--patience', type=int, default=5,
                         help='Number of epochs to wait for improvement before early stopping (default: %(default)s)')
+    parser.add_argument('--normalize', action='store_true',
+                        help='Whether the word embeddings divide by `float(np.linalg.norm(vector) + 1e-6)` (default: %(default)s)')
```

Comment:
this description isn't good. "vector" is unclear here and it's not good to have the code np.xxx in the help.
I think we should just say
Whether the embeddings of each word is normalized to a unit vector